Automattic/wp-calypso
# Help Center: Fix Contact form button styling

Fixes: https://github.com/Automattic/dotcom-forge/issues/8411

## Proposed Changes

* Fixes the button styling when user hovers in disabled and font color when enabled.

Before ( notice hover state when disabled ) :


https://github.com/user-attachments/assets/8354c0c8-c588-4e78-a4d6-233f1b93be07

After: 


https://github.com/user-attachments/assets/a14f1294-9014-444c-915e-8fde1f48fd41



## Why are these changes being made?


* Fix hover state, and remove unnecessary styling.

## Testing Instructions



* Checkout branc or use live link
* Log in with free user
* Open the Help Center, click "Still need help?"
* Hover over the "Continue" button in a disabled state and verify no styling is applied
* Fill out the contact form and hover over the "Continue" button when its enabled to verify the button background gets darker and the font color is still white (you can compare against other similar buttons `/home`)

## Pre-merge Checklist
